### PR TITLE
resource-control: add dashboard panel

### PR DESCRIPTION
**Pull request by HuSharp**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a new article about TiDB Dashboard Resource Control panel.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13907
- Other reference link(s):

### CODE REVIEW

File: tidb-resource-control.md [hunk 1]
```diff
@@ -216,6 +216,8 @@ TiDB regularly collects runtime information about resource control and provides
 
 TiKV also records the request QPS from different resource groups. For more details, see [TiKV Monitoring Metrics Detail](/grafana-tikv-dashboard.md#grpc).
 
+You can view the data of resource groups in the current [`RESOURCE_GROUPS`](/information-schema/information-schema-resource-groups.md) table in TiDB Dashboard. See the [TiDB Dashboard Resource Control page](/dashboard/dashboard-resource-control.md) for details.
```
Comment (Oreoxmt) [inline tidb-resource-control.md:219]:
```suggestion
You can view the data of resource groups in the current [`RESOURCE_GROUPS`](/information-schema/information-schema-resource-groups.md) table in TiDB Dashboard. For more details, see [Resource Manager page](/dashboard/dashboard-resource-control.md).
```

File: dashboard/dashboard-cluster-info.md [hunk 2]
```diff
@@ -12,11 +12,9 @@ On the cluster information page, you can view the running status of TiDB, TiKV,
 
 You can use one of the following two methods to access the cluster information page:
 
-- After logging into TiDB Dashboard, click **Cluster Info** on the left navigation menu:
+* After logging into TiDB Dashboard, click **Cluster Info** on the left navigation menu:
```
Comment (Oreoxmt) [inline dashboard/dashboard-cluster-info.md:15]:
```suggestion
* After logging into TiDB Dashboard, click **Cluster Info** in the left navigation menu.
```

File: TOC.md [hunk 3]
```diff
@@ -945,6 +945,7 @@
         - [Use Diagnostics](/dashboard/dashboard-diagnostics-usage.md)
       - [Monitoring Page](/dashboard/dashboard-monitoring.md)
       - [Search Logs Page](/dashboard/dashboard-log-search.md)
+      - [Resource Control Page](/dashboard/dashboard-resource-control.md)
```
Comment (Oreoxmt) [inline TOC.md:948]:
```suggestion
      - [Resource Manager Page](/dashboard/dashboard-resource-control.md)
```

File: dashboard/dashboard-resource-control.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+title: TiDB Dashboard Resource Control Page
+summary: Introduces how to use TiDB Dashboard's Resource Control page to view information about resource control, so you can estimate cluster capacity and better allocate resources.
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:3]:
```suggestion
title: TiDB Dashboard Resource Manager Page
summary: Introduce how to use the Resource Manager page in TiDB Dashboard to view information about resource control, so you can estimate cluster capacity before resource planning and allocate resources more effectively.
```

File: dashboard/dashboard-resource-control.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+# TiDB Dashboard Resource Control Page
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:6]:
```suggestion
# TiDB Dashboard Resource Manager Page
```

File: dashboard/dashboard-resource-control.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+To implement resource isolation using the [Resource Control](/tidb-resource-control.md) feature, cluster administrators can define Resource Groups and limit read and write quotas through Resource Groups. Before resource planning, you need to know the overall capacity of the cluster. This page can help you view information about resource control so that you can estimate the cluster capacity and better allocate resources.
+
+## Access Page
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:10]:
```suggestion
## Access the page
```
Use sentence case for H2

File: dashboard/dashboard-resource-control.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+    You can select a time range of 10 minutes to 24 hours for the estimation. The time zone is the same as the time zone of the front-end user.
+
+    - When the time window range does not fall between 10 minutes and 24 hours, an error occurs, an error will be reported `ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s`.
+
+    - When the workload within the time window is too low, an error will be reported `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead`.
+
+  You can select the appropriate time range by using **CPU Usage** in [Monitoring Metrics](#monitoring-metrics).
+
+## Monitoring Metrics
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:64]:
```suggestion
## Metrics
```

File: dashboard/dashboard-resource-control.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+You can use one of the following two methods to access the cluster resource control page:
+
+* After logging into TiDB Dashboard, click **Resource Manager** in the left navigation bar.
+
+* Visit <http://127.0.0.1:2379/dashboard/#/resource_manager> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+## Resource Control Details
+
+The Resource Control Details page is shown in the following image:
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:20]:
```suggestion
The following figure shows the Resource Manager details page:
```

File: dashboard/dashboard-resource-control.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+- Metrics: By observing the metrics on the panel, you can understand the current resource consumption status of the cluster as a whole.
+
+## Capacity estimation
+
+Before doing resource planning, you need to know the overall capacity of the cluster. Two estimation methods are currently provided to estimate the capacity of the current cluster's [`Request Unit (RU)`](/tidb-resource-control.md#what-is-request-unit-ru#what-is-request-unit-ru):
+
+- [Estimate capacity based on hardware deployment](/sql-statements/sql-statement-calibrate-resource.md#estimate-capacity-based-on-hardware-deployment)
+    
+    Here accepts the following different workload types:
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:41]:
```suggestion
    TiDB accepts the following different workload types:
```

File: dashboard/dashboard-resource-control.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+    
+    - `tpcc`: applies to workloads with heavy data write. It is estimated based on a workload model similar to `TPC-C`.
+    - `oltp_write_only`: applies to workloads with heavy data write. It is estimated based on a workload model similar to `sysbench oltp_write_only`.
+    - `oltp_read_write`: applies to workloads with even data read and write. It is estimated based on a workload model similar to `sysbench oltp_read_write`.
+    - `oltp_read_only`: applies to workloads with heavy data read. It is estimated based on a workload model similar to `sysbench oltp_read_only`.
+
+    ![Estimated capacity based on hardware deployment](/media/dashboard/dashboard-resource-manager-calibrate-by-hardware.png)
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:48]:
```suggestion
    ![Calibrate by Hardware](/media/dashboard/dashboard-resource-manager-calibrate-by-hardware.png)
```

File: dashboard/dashboard-resource-control.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+    Total RU of user resource groups indicates the total number of RUs currently available except for `default` users. The system will alert when this value is less than the capacity estimate. TiDB automatically creates a `default` resource group during cluster initialization. For this resource group, the default value of `RU_PER_SEC` is `UNLIMITED`. When all users belong to the `default` resource group, resources are allocated in the same way as when resource control is turned off.
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:50]:
```suggestion
    The **Total RU of user resource groups** represents the total amount of RU for all users resource groups, excluding the `default` user. If this value is less than the estimated capacity, the system triggers an alert. By default, the system allocates unlimited usage to the predefined `default` resource group. When all users belong to the `default` resource group, resources are allocated in the same way as when resource control is turned off.
```

File: dashboard/dashboard-resource-control.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+    ![Calibrate by Workload](/media/dashboard/dashboard-resource-manager-calibrate-by-workload.png)
+
+    You can select a time range of 10 minutes to 24 hours for the estimation. The time zone is the same as the time zone of the front-end user.
+
+    - When the time window range does not fall between 10 minutes and 24 hours, an error occurs, an error will be reported `ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s`.
+
+    - When the workload within the time window is too low, an error will be reported `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead`.
+
+  You can select the appropriate time range by using **CPU Usage** in [Monitoring Metrics](#monitoring-metrics).
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:62]:
```suggestion
    You can select a time range for estimation within the range of 10 minutes to 24 hours. The time zone used is the same as that of the front-end user.

    - When the time window range does not fall between 10 minutes and 24 hours, an error `ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s` is displayed.

    - When the workload within the time window is too low, an error `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead` is displayed.

  You can select an appropriate time range using **CPU Usage** in the [Metrics](#monitoring-metrics) section.
```

File: dashboard/dashboard-resource-control.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+
+- Total RU Consumed: The total consumption of Request Units counted in real time
+- RU Consumed by Resource Groups: The number of Request Units consumed by resource groups in real time.
+- TiDB
+    - CPU Quota: The maximum CPU usage of TiDB
+    - CPU Usage: CPU usage of all TiDB instances
+- TiKV
+    - CPU Quota: The maximum CPU usage of TiKV
+    - CPU Usage: CPU usage of all TiKV instances
+    - IO MBps: I/O throughput of all TiKV instances
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:76]:
```suggestion
By observing the metrics on the panels, you can understand the current overall resource consumption status of the cluster. The monitoring metrics and their meanings are as follows:

- Total RU Consumed: The total consumption of Request Units counted in real time.
- RU Consumed by Resource Groups: The number of Request Units consumed by resource groups in real time.
- TiDB
    - CPU Quota: The maximum CPU usage of TiDB.
    - CPU Usage: The total CPU usage of all TiDB instances.
- TiKV
    - CPU Quota: The maximum CPU usage of TiKV.
    - CPU Usage: The total CPU usage of all TiKV instances.
    - IO MBps: The total I/O throughput of all TiKV instances.
```

File: dashboard/dashboard-resource-control.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+summary: Introduces how to use TiDB Dashboard's Resource Control page to view information about resource control, so you can estimate cluster capacity and better allocate resources.
+---
+
+# TiDB Dashboard Resource Control Page
+
+To implement resource isolation using the [Resource Control](/tidb-resource-control.md) feature, cluster administrators can define Resource Groups and limit read and write quotas through Resource Groups. Before resource planning, you need to know the overall capacity of the cluster. This page can help you view information about resource control so that you can estimate the cluster capacity and better allocate resources.
+
+## Access Page
+
+You can use one of the following two methods to access the cluster resource control page:
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-control.md:12]:
```suggestion
You can use one of the following two methods to access the Resource Manager page:
```

File: dashboard/dashboard-resource-control.md [hunk 15]
```diff
@@ -0,0 +1,76 @@
+---
+title: TiDB Dashboard Resource Manager Page
+summary: Introduce how to use the Resource Manager page in TiDB Dashboard to view the information about resource control, so you can estimate cluster capacity before resource planning and allocate resources more effectively.
+---
+
+# TiDB Dashboard Resource Manager Page
+
+To implement resource isolation using the [Resource Control](/tidb-resource-control.md) feature, cluster administrators can create resource groups and set read and write quotas for each group. Before resource planning, you need to know the overall capacity of the cluster. This document helps you view information about resource control, so you can estimate the cluster capacity before resource planning and allocate resources more effectively.
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:8]:
```suggestion
To implement resource isolation using the [Resource Control](/tidb-resource-control.md) feature, cluster administrators can create resource groups and set read and write quotas for each group. Before resource planning, you need to know the overall capacity of the cluster. This document helps you view the information about resource control, so you can estimate the cluster capacity before resource planning and allocate resources more effectively.
```

File: dashboard/dashboard-resource-control.md [hunk 16]
```diff
@@ -0,0 +1,76 @@
...
+
+## Access the page
+
+You can use one of the following two methods to access the Resource Manager page:
+
+* After logging in to TiDB Dashboard, click **Resource Manager** in the left navigation menu.
+
+* Visit <http://127.0.0.1:2379/dashboard/#/resource_manager> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
+
+## Resource Manager details page
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:18]:
```suggestion
## Resource Manager details page
```
```suggestion
## Resource Manager page
```

File: dashboard/dashboard-resource-control.md [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+
+The following figure shows the Resource Manager details page:
+
+![TiDB Dashboard: Resource Manager](/media/dashboard/dashboard-resource-manager-info.png)
+
+The Resource Manager details page contains the following three sections:
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:24]:
```suggestion
The Resource Manager page contains the following three sections:
```

File: dashboard/dashboard-resource-control.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
...
+
+- Configuration: This section displays data obtained from the `RESOURCE_GROUPS` table of TiDB, containing information about all resource groups. For more information, see [`RESOURCE_GROUPS`](/information-schema/information-schema-resource-groups.md).
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:26]:
```suggestion
- Configuration: This section displays the data obtained from the `RESOURCE_GROUPS` table of TiDB. It contains the information about all resource groups. For more information, see [`RESOURCE_GROUPS`](/information-schema/information-schema-resource-groups.md).
```

File: dashboard/dashboard-resource-control.md [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+
+- Metrics: By observing the metrics on the panel, you can understand the current overall resource consumption status of the cluster.
+
+## Estimate Capacity
+
+Before resource planning, you need to know the overall capacity of the cluster. TiDB provides two methods to estimate the capacity of [Request Unit (RU)](/tidb-resource-control.md#what-is-request-unit-ru#what-is-request-unit-ru) in the current cluster:
+
+- [Estimate capacity based on hardware deployment](/sql-statements/sql-statement-calibrate-resource.md#estimate-capacity-based-on-hardware-deployment)
+    
+    TiDB accepts the following different workload types:
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:41]:
```suggestion
    TiDB accepts the following workload types:
```

File: dashboard/dashboard-resource-control.md [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+
+- [Estimate capacity based on actual workload](/sql-statements/sql-statement-calibrate-resource.md#estimate-capacity-based-on-actual-workload)
+
+    ![Calibrate by Workload](/media/dashboard/dashboard-resource-manager-calibrate-by-workload.png)
+
+    You can select a time range for estimation within the range of 10 minutes to 24 hours. The time zone used is the same as that of the front-end user.
+
+    - When the time window range does not fall between 10 minutes and 24 hours, an error `ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s` is displayed.
+
+    - When the workload within the time window is too low, an error `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead` is displayed.
```
Comment (hfxsd) [inline dashboard/dashboard-resource-control.md:60]:
```suggestion
    - When the time window range does not fall between 10 minutes and 24 hours, the following error is displayed `ERROR 1105 (HY000): the duration of calibration is too short, which could lead to inaccurate output. Please make the duration between 10m0s and 24h0m0s`.

    - When the workload within the time window is too low, the following error is displayed `ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead` is displayed.
```

**Review by hfxsd — APPROVED** [review]

File: dashboard/dashboard-cluster-info.md [hunk 21]
```diff
@@ -12,11 +12,9 @@ On the cluster information page, you can view the running status of TiDB, TiKV,
 
 You can use one of the following two methods to access the cluster information page:
 
-- After logging into TiDB Dashboard, click **Cluster Info** on the left navigation menu:
+* After logging into TiDB Dashboard, click **Cluster Info** in the left navigation menu.
```
Comment (Oreoxmt) [inline dashboard/dashboard-cluster-info.md:15]:
```suggestion
* After logging in to TiDB Dashboard, click **Cluster Info** in the left navigation menu.
```

File: dashboard/dashboard-resource-manager.md [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+The Resource Manager page contains the following three sections:
+
+- Configuration: This section displays the data obtained from the `RESOURCE_GROUPS` table of TiDB. It contains the information about all resource groups. For more information, see [`RESOURCE_GROUPS`](/information-schema/information-schema-resource-groups.md).
+
+- Estimate Capacity: Before resource planning, you need to know the overall capacity of the cluster. You can use one of the following methods:
+
+    - [Estimate capacity based on actual workload](/sql-statements/sql-statement-calibrate-resource.md#estimate-capacity-based-on-actual-workload)
+    - [Estimate capacity based on hardware deployment](/sql-statements/sql-statement-calibrate-resource.md#estimate-capacity-based-on-hardware-deployment)
+
+- Metrics: By observing the metrics on the panel, you can understand the current overall resource consumption status of the cluster.
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-manager.md:33]:
```suggestion
- Metrics: By observing the metrics on the panels, you can understand the current overall resource consumption status of the cluster.
```

File: dashboard/dashboard-resource-manager.md [hunk 23]
```diff
@@ -0,0 +1,76 @@
...
+    TiDB accepts the following workload types:
+    
+    - `tpcc`: applies to workloads with heavy data write. It is estimated based on a workload model similar to `TPC-C`.
+    - `oltp_write_only`: applies to workloads with heavy data write. It is estimated based on a workload model similar to `sysbench oltp_write_only`.
+    - `oltp_read_write`: applies to workloads with even data read and write. It is estimated based on a workload model similar to `sysbench oltp_read_write`.
+    - `oltp_read_only`: applies to workloads with heavy data read. It is estimated based on a workload model similar to `sysbench oltp_read_only`.
+
+    ![Calibrate by Hardware](/media/dashboard/dashboard-resource-manager-calibrate-by-hardware.png)
+
+    The **Total RU of user resource groups** represents the total amount of RU for all users resource groups, excluding the `default` user. If this value is less than the estimated capacity, the system triggers an alert. By default, the system allocates unlimited usage to the predefined `default` resource group. When all users belong to the `default` resource group, resources are allocated in the same way as when resource control is disabled.
```
Comment (Oreoxmt) [inline dashboard/dashboard-resource-manager.md:50]:
```suggestion
    The **Total RU of user resource groups** represents the total amount of RU for all user resource groups, excluding the `default` resource group. If this value is less than the estimated capacity, the system triggers an alert. By default, the system allocates unlimited usage to the predefined `default` resource group. When all users belong to the `default` resource group, resources are allocated in the same way as when resource control is disabled.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: dashboard/dashboard-cluster-info.md (hunk 21 above)
Comment (HuSharp) [reply dashboard/dashboard-cluster-info.md]: Okay, change all `into` to `in to` in all dashboard docs.

File: dashboard/dashboard-resource-manager.md [hunk 24]
```diff
@@ -0,0 +1,76 @@
+---
+title: TiDB Dashboard Resource Manager Page
+summary: Introduce how to use the Resource Manager page in TiDB Dashboard to view the information about resource control, so you can estimate cluster capacity before resource planning and allocate resources more effectively.
+---
+
+# TiDB Dashboard Resource Manager Page
+
+To implement resource isolation using the [Resource Control](/tidb-resource-control.md) feature, cluster administrators can create resource groups and set read and write quotas for each group. Before resource planning, you need to know the overall capacity of the cluster. This document helps you view the information about resource control, so you can estimate the cluster capacity before resource planning and allocate resources more effectively.
```
[the remark this answers is not in the document]
Comment (songrijie) [reply dashboard/dashboard-resource-manager.md:8]: "read and write quotas" is a little confused, as we only accept a unified quota for both read and write.

**Review by songrijie — APPROVED** [review]
REST LGTM

**Pull request merged**